csrankings-d.csv

```diff
@@ -904,6 +904,7 @@ Despina Polemi,University of Piraeus,https://www.unipi.gr/unipi/en/dpolemi.html,
 Despoina Polemi,University of Piraeus,https://www.unipi.gr/unipi/en/dpolemi.html,WrDuv4kAAAAJ
 Detlef Krömker,Goethe University Frankfurt,https://www.gdv.informatik.uni-frankfurt.de/index.php?m=5&sm=2&c=%2Fteam%2Fdetails%2Fkroemker.php,NOSCHOLARPAGE
 Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det,nh6AHbIAAAAJ
+Detmar Meurers,University of Tübingen,http://www.sfs.uni-tuebingen.de/~dm/,XOs7l48AAAAJ

```

Comment:
In Linguistics; please provide evidence of meeting inclusion criteria.

Reply:
> In Linguistics; please provide evidence of meeting inclusion criteria.

You just need to check [the professor's google scholar page](https://scholar.google.de/citations?user=XOs7l48AAAAJ&hl=de), you can clearly see he deals with NLP and Computational Linguistics.

Reply:
In fact, his description goes "Professor of Computational Linguistics" (NLP, in other words)

Reply:
That's not sufficient. They need to be part of a CS department or there needs to be evidence of the latter point below:

> Eligible faculty include only full-time, tenure-track research faculty members on a given campus who can solely advise PhD students in Computer Science. Faculty not in a CS department or similar who can advise PhD students in CS can be included regardless of their home department.

Reply:
Bump @Giulio-Cusenza